site/docs/introduction/aggregate-programming.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Aggregate Programming 
+---
+
+# The Challenge of Maintaining Distributed Systems 
+
+The **progressive increase** in the number of interconnected devices, leads to a **rise in the costs** associated with maintaining distributed systems. This creates **significant challenges** in implementing software services on a **global scale** using the traditional approach of individually programming each agent. This situation drives the search for solutions aimed at improving the **autonomy** of computing systems and **reducing their complexity**.
+
+## Aggregate Programming as a Relevant Approach
+
+In this context, **aggregate programming** emerges as a relevant approach. It is based on the **functional composition** of **reusable collective behavior blocks**, with the goal of efficiently achieving **complex** and **resilient behaviors** in **dynamic networks**.
+
+### Key Properties of Aggregate Programming
+
+- **Self-stabilization**
+- **Independence from density**
+
+These properties are initially studied on **basic building blocks**, such as:
+- **Broadcasting**
+- **Distance estimation**
+- **Data aggregation**
+
+These building blocks are then transferred and validated on more **complex systems** created through **functional composition**.
+
+Aggregate programming finds its **conceptual roots** in **Field Calculus** and presents itself as a minimal functional programming language designed for the specification and composition of collective behaviors.
+
+## Levels of Abstraction in Aggregate Programming
+
+Through multiple levels of abstraction[<sup>1</sup>](#bibliography), aggregate programming mitigates the complexity involved in distributed coordination within IoT network environments:
+
+<div className="centered">
```

Comment:
try to avoid using the HTML syntax, try mdx and react's components if it is possible, I think that docusaurus supports them, but double check.